<?xml version="1.0"?>
<launch>
  <arg name="mav_name" default="firefly"/>
  <arg name="use_unity_editor" default="true" />

  <group ns="$(arg mav_name)" >

    <!-- RPG Flightmare Unity Render. -->
    <node pkg="flightrender" type="RPG_Flightmare.x86_64" name="rpg_flightmare_render" unless="$(arg use_unity_editor)">
    </node>

    <node pkg="flightros" type="flight_pilot_node" name="flight_pilot_node" output="screen">
      <rosparam file="$(find flightros)/params/default.yaml" /> 
      <remap from="flight_pilot/state_estimate" to="ground_truth/odometry" />
    </node>

  </group> 
  
  <!-- Static transform from base_link to unity_cam -->
  <node pkg="tf" type="static_transform_publisher" name="link_cam_unity" args="0 0 0 3.14 3.14 0 /$(arg mav_name)/base_link unity_cam 100" />

</launch>
